docs/DOCS_README.md

```diff
@@ -1,4 +1,8 @@
-# Docs Build Workflow
+## Updating the docs
+
+If you want to open a PR on the Cosmos SDK to update the documentation, please follow the guidelines in the [`CONTRIBUTING.md`](https://github.com/cosmos/cosmos-sdk/tree/master/CONTRIBUTING.md)
+
+## Docs Build Workflow
```

Comment:
this should be H1 and `Updating the Docs` can be H2 and the rest of the headers can stay H2